src/ui/server/controllers/insurance/account/sign-in/index.test.ts

```diff
@@ -287,6 +287,20 @@ describe('controllers/insurance/account/sign-in', () => {
           expect(res.redirect).toHaveBeenCalledWith(`${SUSPENDED_ROOT}?id=${accountSignInResponse.accountId}`);
         });
       });
+
+      describe('when password only contains numbers', () => {
+        it('should call api.keystone.account.signIn with password as a string', async () => {
+          req.body[PASSWORD] = 12345;
```

Comment:
This would come through as a string from the HTML form

Reply:
If this is updated to be a string like it would be outside of tests, this test will fail

Reply:
I will address this and maybe able to simplify the E2E test in my next PR 👍